src/spec-common/injectHeadless.ts

```diff
@@ -525,24 +536,18 @@ async function runLifecycleCommand({ lifecycleHook }: ResolverParameters, contai
 					return runSingleCommand(command, name);
 				});
 			}
+			infoOutput.event({
```

Comment:
This event needs to move to right after `await Promise.all(commands);`.

Reply:
Done.